pkg/local_object_storage/writecache/flush.go

```diff
@@ -125,7 +125,7 @@ func (c *cache) flushBigObjects() {
 					return nil
 				}
 
-				if _, err := c.blobstor.PutRaw(addr, data); err != nil {
+				if _, err := c.blobstor.PutRaw(addr, data, true); err != nil {
```

Comment:
I haven't yet found the right way to make a decision here.
So we either
1. Unmarshal object just to determine if it needs to be compressed. This is slow.
2. Put compressed objects in writecache. This needs some refactoring, I think I will implement it right here.

Reply:
or since we put objects to writecache in "unmarshaled" format we can
3. Calc compression flag in the same way and keep its value in writecache. Later we can pass the flag during `flush`.

Reply:
Yes, 3 should be faster. Fixed.